app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx

```diff
@@ -87,7 +105,11 @@ export const ListingDetailsAdditionalInformation = ({
                 <div className={"flex mt-4"}>
                   <span className={"font-bold mr-1"}>{t("listings.realtorCommissionHowTo")}</span>
                   <span>
```

Comment:
I know this is out of the scope for this story, but if it's a quick fix can we include it? 
![image](https://user-images.githubusercontent.com/98352600/208800016-f61a6a7f-2d26-484e-bb99-3b020c75bd90.png)

Reply:
> I know this is out of the scope for this story, but if it's a quick fix can we include it? ![image](https://user-images.githubusercontent.com/98352600/208800016-f61a6a7f-2d26-484e-bb99-3b020c75bd90.png)

@christine-sfg Got a fix in! Mind giving it a look before I merge?

Reply:
looks great! thanks for squeezing this in!